generation/new_client/templates/owlbot.yaml.monorepo.j2

```diff
@@ -17,10 +17,10 @@ deep-remove-regex:
 - "/{{ module_name }}/grpc-google-.*/src"
 - "/{{ module_name }}/proto-google-.*/src"
 - "/{{ module_name }}/google-.*/src"
-
 deep-preserve-regex:
 - "/{{ module_name }}/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
-
+- "/{{ module_name }}/google-.*/src/main/resources/META-INF/native-image/*"
```

Comment:
```suggestion
- "/{{ module_name }}/google-.*/src/main/resources/META-INF/native-image/.*"
```

This is not glob pattern. It's a regular expression.

Reply:
Ah gotcha. Will fix.